tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -518,6 +518,84 @@ def test_invalid_groupby(self):
         assert response.status_code == 400, response.content
         assert response.data['detail'] == 'Invalid groupby value requested. Allowed values are project.id, issue.id'
 
+    def test_non_aggregated_fields_with_groupby(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project()
+        self.store_event(
+            data={
+                'event_id': 'a' * 32,
+                'message': 'how to make fast',
+                'timestamp': self.min_ago,
+            },
+            project_id=project.id
+        )
+
+        with self.feature('organizations:events-v2'):
+            response = self.client.get(
+                self.url,
+                format='json',
+                data={
+                    'field': ['project.id'],
+                    'groupby': ['issue.id'],
+                },
+            )
+        assert response.status_code == 400, response.content
+        assert response.data['detail'] == 'Invalid query.'
+
+    def test_non_existent_params(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project()
+        self.store_event(
+            data={
+                'event_id': 'a' * 32,
+                'message': 'how to make fast',
+                'timestamp': self.min_ago,
+            },
+            project_id=project.id
+        )
+
+        with self.feature('organizations:events-v2'):
+            response = self.client.get(
+                self.url,
+                format='json',
+                data={
+                    # TODO(lb): is it ok if extraneous data is included as long as the minimum
+                    # is met?
```

Comment:
I think ignoring input we don't understand is ok. The alternative is to error out, but we don't error out when arbitrary URL parameters are added elsewhere in the app.